lib/utils.js

```diff
@@ -223,9 +223,10 @@ function prettifyErrorLog ({
  * `undefined` will be returned. Otherwise, a string from the specified
  * `colorizer` is returned.
  */
-function prettifyLevel ({ log, colorizer = defaultColorizer, levelKey = LEVEL_KEY }) {
+function prettifyLevel ({ log, colorizer = defaultColorizer, levelKey = LEVEL_KEY, prettifier }) {
   if (levelKey in log === false) return undefined
-  return colorizer(log[levelKey])
+  const output = log[levelKey]
+  return prettifier ? prettifier(output) : colorizer(output)
```

Comment:
Does this not break colorization?

Reply:
Do you mean colorization via the user-supplied `prettifier` function? My intent is that the user owns any/all formatting (including colorizing) via this function.

I personally want to customize the coloring done via this method to alter it from the default supplied here.

If this is a concern I'm happy to document how to obtain the built-in behavior in the prettifier so users who only wish to lightly customize output can do so.

Reply:
> If this is a concern I'm happy to document how to obtain the built-in behavior in the prettifier so users who only wish to lightly customize output can do so.

That would be acceptable.

Reply:
Added! Thanks so much for your help getting this PR put in order :)